docs/book/src/reference/envtest.md

```diff
@@ -97,12 +106,11 @@ You can use environment variables and/or flags to specify the `kubectl`,`api-ser
 ### Environment Variables
 
 | Variable name | Type | When to use |
-| --- | :--- | :--- |
-| `USE_EXISTING_CLUSTER` | boolean | Instead of setting up a local control plane, point to the control plane of an existing cluster. |
```

Comment:
Does this environment variable no longer get used when using envtest?

Reply:
HUn if it was before is good we leave that, I thought that it was added. @yashsingh74 could you please just revert the changes on the table and leave it as it was?

Reply:
No, it is no longer required, as we mentioned below that how we can use the existing cluster.

Reply:
Done